### PR TITLE
Us116068/delete dialog

### PR DESCRIPTION
**Pull request by BraydonKains**
delete dialog

### CODE REVIEW

File: src/components/d2l-teacher-course-creation-admin.js [hunk 1]
```diff
@@ -83,10 +86,21 @@ class TeacherCourseCreationAdmin extends BaseMixin(LitElement) {
 			});
 	}
 
+	async _deleteAssociation() {
+		if (this.dialogAssociation) {
+			console.log(this.dialogAssociation);
```
Comment (josephsaffron) [inline src/components/d2l-teacher-course-creation-admin.js:91]: debug code?

File: src/components/dialog/delete-dialog.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	async open() {
+		this.deleteDialogOpened = true;
+	}
+
+	_close() {
+		this.deleteDialogOpened = false;
+	}
+
+	_handleDialogClose(event) {
+		if (event.detail.action === 'yes') {
```
Comment (josephsaffron) [inline src/components/dialog/delete-dialog.js:40]: did you want to change the action to match the button text?  You don't have to.

File: src/components/d2l-teacher-course-creation-admin.js (hunk 1 above)
Comment (BraydonKains) [reply src/components/d2l-teacher-course-creation-admin.js:91]: oops

File: src/components/dialog/delete-dialog.js (hunk 2 above)
Comment (BraydonKains) [reply src/components/dialog/delete-dialog.js:40]: Yeah I could, it doesn't matter for any user stuff but for code readability it would be nice

**Pull request merged**